src/main/java/org/example/homework7/RomanNumsCalc/Calculator.java

```diff
@@ -0,0 +1,29 @@
+package org.example.homework7.RomanNumsCalc;
+
+public class Calculator {
+
+    public int calculate(RomanNumber num1, RomanNumber num2, String operation) {
+        int first = num1.getIntValue();
+        int second = num2.getIntValue();
+
+        int result = 0;
+
+        switch (operation) {
+            case "+":
+                result = first + second;
+                break;
+            case "-":
+                result = first - second;
+                break;
+            case "*":
+                result = first * second;
+                break;
+            case "/":
+                result = first / second;
```

Comment:
Division by zero is not handled

Reply:
sorry, i didn't add it, because in task description we didn't have option to divide by zero. because we use set of defined roman numbers, and here wasn't zero value